contracts/migrations/2_mockcoin.js

```diff
@@ -1,7 +1,7 @@
 const MockCoin = artifacts.require("MockCoin");
 
 module.exports = async (deployer, network, accounts) => {
-  if (network == "development" || network == "test") {
+  // if (network == "development" || network == "test") {
     await deployer.deploy(MockCoin);
-  }
+  //}
```

Comment:
We are going to use an already existing ERC20 token. We should not deploy a new one to mainnet. It is already there!